src/catkin_pkg/changelog.py

```diff
@@ -53,6 +53,8 @@
 import pkg_resources
 import re
 
+from io import open
```

Comment:
Is there a reason to prefer decoding after the fact other than that's what's already done elsewhere? As far as I can tell we never _want_ to work with the unencoded file contents.